src/pages/progetto.yaml

```diff
@@ -173,403 +173,59 @@ components:
+      subtitle: I princìpi e i valori che ispirano il nostro lavoro e i nostri strumenti #C
       buttons: #C #I
-        - label : Scopri lorem ipsum #C
+        - label : Scopri visione e missione #C
           btnStyle: primary
-          url: "lorem-ipsum/" #M
+          url: "visione-e-missione/" #M
 
     # —
     # HIGHLIGHT 
     # —
-    - title : Lorem ipsum #C
+    - title : Formazione e disseminazione #C
       id: HL2
       img : /images/400x300.png #I
       alt: Alt lorem ipsum #C
       headingLevel : 3
-      background: null  # null | light | medium | dark  
+      background: light  # null | light | medium | dark  
       big: false
       specular: true
-      subtitle: Lorem ipsum #C 
+      subtitle: Il nostro impegno per diffondere e condividere con la community le buone pratiche di design applicate ai servizi digitali pubblici #C 
       numbers: #C #I
         items:
         - icon: sprites.svg#it-pa
           number: 555
-          label: Number label
+          label: Corsi erogati
         - icon: sprites.svg#it-user
           number: 555
-          label: Number label
-      text: Lorem ipsum #C
+          label: Video realizzati
+      text: Abbiamo l’obiettivo di divulgare i princìpi, la cultura della progettazione e le risorse di Designers Italia, per favorire la condivisione di buone pratiche tra progettisti di servizi pubblici digitali, community del design e mondo della PA #C
       buttons: #C #M
-        - label : Scopri lorem ipsum 
+        - label : Scopri formazione e disseminazione
           btnStyle: primary
-          url: "lorem-ipsum/" 
-        - label : Label button
-          btnStyle: outline-secondary
-          url: "#"
-        - label : Label button
-          btnStyle: outline-secondary
-          url: "#"
-    
+          url: "formazione-e-disseminazione/" 
+     
     # —
     # HIGHLIGHT BIG IMG
     # —
-    - title : Lorem ipsum #C
+    - title : Storia e progetti #C
       id: HL3
       img : /images/400x300.png #I
       alt: Alt lorem ipsum #C
       headingLevel : 3
       background: medium # null | light | medium | dark  
       big: true
       specular: true
-      subtitle: Lorem ipsum #C
-      buttons: #C #M
-        - label : Scopri lorem ipsum 
-          btnStyle: primary
-          url: "lorem-ipsum/" 
-
-    # —
-    # HIGHLIGHT
-    # —
-    - title : Lorem ipsum #C
-      id: HL4
-      img : /images/400x300.png #I
-      alt: Alt lorem ipsum #C
-      headingLevel : 3
-      big: false
-      background: dark
-      specular: true
-      subtitle: Lorem ipsum #C
-      numbers: #C #I
-        items:
-        - icon: sprites.svg#it-pa
-          number: 555
-          label: Number label
-        - icon: sprites.svg#it-user
-          number: 555
-          label: Number label
-      text: Lorem ipsum #C
+      subtitle: Una storia fatta di persone, idee e open design: le piattaforme, gli strumenti e i progetti a cui Designers Italia ha contribuito  #C
```

Comment:
@danielaiozzo c'è un "due punti" `:` a metà frase qui riga 227... per questo fallisce la build. Buildandolo in locale mi fa vedere l'errore

![error building](https://user-images.githubusercontent.com/6736103/203384932-38b65a04-1ec9-4968-8076-c73e3c3c5df6.png)